libraries/AP_BattMonitor/AP_BattMonitor_UAVCAN.cpp

```diff
@@ -85,6 +88,17 @@ void AP_BattMonitor_UAVCAN::subscribe_msgs(AP_UAVCAN* ap_uavcan)
         AP_BoardConfig::allocation_error("UAVCAN Mppt::Stream subscriber start problem");
         return;
     }
+
+    const uint8_t driver_index = ap_uavcan->get_driver_index();
+    outputEnable_client[driver_index] = new uavcan::ServiceClient<mppt::OutputEnable>(*node);
+    if (outputEnable_client[driver_index] == nullptr) {
+        AP_HAL::panic("AP_UAVCAN_DNA: outputEnable_client[%d]", driver_index);
+    }
+    int res = outputEnable_client[driver_index]->init();
```

Comment:
should this be an int?

Reply:
bit of a nitpick but you could collapse these two lines and remove the local variable, "res".

Reply:
done